samples/WebApp/MassTransit/TestCourier.cs

```
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using MassTransit.Courier;
+
+namespace WebApp.MassTransit
+{
+    public class DownloadImageActivity : IActivity<DownloadImageArguments, DownloadImageLog>
+    {
+        public async Task<ExecutionResult> Execute(ExecuteContext<DownloadImageArguments> context)
+        {
+            return default;
+        }
+
+        public async Task<CompensationResult> Compensate(CompensateContext<DownloadImageLog> context)
+        {
+            return default;
+        }
+    }
+
+    public class FilterImageActivity : IActivity<FilterImageArguments, FilterImageLog>
+    {
+        public async Task<ExecutionResult> Execute(ExecuteContext<FilterImageArguments> context)
+        {
+            return default;
```

Should throw an exception here in order to run compensate activity